extensions/wikia/ExactTargetUpdates/tasks/ExactTargetUserUpdateDriver.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Wikia\ExactTarget;
+
+class ExactTargetUserUpdateDriver {
+
+	/**
+	 * Allow to create when new email exists and is different than old one
+	 * @param string $oldEmail Current email from ExactTarget database
+	 * @param string $newEmail New email from Wikia to be updated
+	 * @return bool
+	 */
+	public static function shouldCreateAsEmailChanged( $oldEmail, $newEmail ) {
```

Comment:
this and `shouldDeleteAsEmailChanged` have same body logic. difference is `!empty(oldEmail)` and `!empty(newEmail)`.
Consider replaced by one function. `isEmailChanged`?